app/models/admin/action_log_filter.rb

```diff
@@ -95,6 +103,9 @@ def scope_for(key, value)
     when 'target_account_id'
       account = Account.find_or_initialize_by(id: value)
       latest_action_logs.where(target: [account, account.user].compact)
+    when 'target_domain'
+      normalized_domain = TagManager.instance.normalize_domain(value)
+      latest_action_logs.where(human_identifier: normalized_domain, target_type: INSTANCE_TARGET_TYPES)
```

Comment:
I've not done any indexing on human_identifier, since it'd still need the index created concurrently and take potentially a sizable amount of resources.

Reply:
`Admin::ActionLogFilter` doesn't appear to have any test coverage yet.

Reply:
> I've not done any indexing on human_identifier

And did you test how it performs? It should be easily possible to generate a large amount of action logs to see how it deals with that.

> since it'd still need the index created concurrently and take potentially a sizable amount of resources.

Which (I think) would be OK in a post deployment migration.

Reply:
I mean, do we want to add an index on human_readable, or back fill route_param and index there? I think the latter is probably be better option?

Reply:
It's also only searching the latest 5 that match the query, so hopefully not too expensive?

Reply:
My thoughts exactly. Just to be on the safe side, I just tested this with 100,000 records. It is still really fast. I am pretty sure we can live without the index for now and further optimize this later if the need arises.